docs/migrating_to_6.md

```diff
@@ -90,7 +96,11 @@ await mongoose.connect('mongodb://localhost:27017/test', {
 });
 ```
 
-<h3 id="the-aspromise-method-for-connections"><a href="#the-aspromise-method-for-connections">The `asPromise()` Method for Connections</a></h3>
+<h3 id="the-aspromise-method-for-connections"><a href="#the-aspromise-method-for-connections">
+
+The `asPromise()` Method for Connections
```

Comment:
Why add these newlines? Should be able to just replace

```
`asPromise()`
```

with:

```
<code>asPromise()</code>
```

Reply:
in that case, i had just opted to change to new-lines and not code, because there were many instances, but i will try to update it

Reply:
updated in c34c0767eab4d5d46b414514515405c5c3b6bd84